src/rpc/blockchain.cpp

```diff
@@ -126,94 +126,6 @@ UniValue blockheaderToJSON(const CBlockIndex* blockindex)
     return result;
 }
 
-// insightexplorer
-UniValue blockToDeltasJSON(const CBlock& block, const CBlockIndex* blockindex)
```

Comment:
We only just reviewed this as a separate function in #3953; please leave it where it was so this PR doesn't have the additional burden of reviewing moved code.